modules/core/02-client/client/cli/tx.go

```diff
@@ -21,6 +22,8 @@ import (
 	"github.com/cosmos/ibc-go/v7/modules/core/exported"
 )
 
+const FlagAuthority = "authority"
```

Comment:
does SDK have a flag for this we can reference or easier to use our own?

Reply:
ah, not that I could find. This mimics usage of `auth`'s cli, can re-use this with Charly's PR.

Reply:
https://github.com/cosmos/ibc-go/pull/4522#discussion_r1311349214